src/ray/raylet/raylet.cc

```diff
@@ -33,10 +33,6 @@ static const std::vector<std::string> node_manager_message_enum =
     GenerateEnumNames(ray::protocol::EnumNamesMessageType(),
```

Comment:
should the `node_manager_message_enum` be removed as well?

Reply:
raylet client hasn't changed, would remove it when change raylet client